integration-cli/docker_cli_ps_test.go

```diff
@@ -734,3 +734,42 @@ func (s *DockerSuite) TestPsNotShowPortsOfStoppedContainer(c *check.C) {
 	fields = strings.Fields(lines[1])
 	c.Assert(fields[len(fields)-2], checker.Not(checker.Equals), expected, check.Commentf("Should not got %v", expected))
 }
+
+func (s *DockerSuite) TestPsShowMounts(c *check.C) {
+	dockerCmd(c, "volume", "create", "--name", "ps-volume-test")
+	dockerCmd(c, "run", "--name=volume-test-1", "-d", "--volume", "ps-volume-test:/test", "busybox", "top")
+	c.Assert(waitRun("volume-test-1"), checker.IsNil)
+	dockerCmd(c, "run", "--name=volume-test-2", "-d", "--volume", "/data", "busybox", "top")
+	c.Assert(waitRun("volume-test-2"), checker.IsNil)
+
+	out, _ := dockerCmd(c, "ps", "--format", "{{.Names}} {{.Mounts}}")
+
+	lines := strings.Split(strings.TrimSpace(string(out)), "\n")
+	c.Assert(lines, checker.HasLen, 2)
+
+	fields := strings.Fields(lines[0])
+	c.Assert(fields, checker.HasLen, 2)
+
+	annonymounsVolumeID := fields[1]
+
+	fields = strings.Fields(lines[1])
+	c.Assert(fields[1], checker.Equals, "ps-volume-test")
+
+	// filter by volume name
+	out, _ = dockerCmd(c, "ps", "--format", "{{.Names}} {{.Mounts}}", "--filter", "volume=ps-volume-test")
```

Comment:
Can you dup these two --filter runs with a filter that doesn't match just so we can verify that the result list is empty?